### PR TITLE
Dockerise

### PR DESCRIPTION
**Pull request by evanjt**
This PR sets the same objectives as the ones defined in the Nomis portal PR: https://github.com/RIVER-EPFL/nomis-data-portal/pull/1

### CODE REVIEW

**Review by sbancal — COMMENTED** [review]
Hey,

I have the same comments as for https://github.com/RIVER-EPFL/nomis-data-portal/pull/1
😉

File: app/start.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# RShiny deployments do not pass system environment variables
+# to the application. This script grabs the env vars set in docker
+# and creates a config file which is read by the application on
+# initialisation, then starts the application
+
+# Delete any existing app config file and recreate from ENV vars
+rm -f secrets.R
```
Comment (sbancal) [inline app/start.sh]: not needed, since line 12 will overwrite the file.

File: app/start.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+MY_DB_HOST <- '$METALP_DB_HOSTNAME'
+MY_DB_PORT <- $METALP_DB_PORT
+MY_DB_USER <- '$METALP_DB_USERNAME'
+MY_DB_PWD <- '$METALP_DB_PASSWORD'
+NOREPLY_ADDRESS <- '$METALP_NOREPLY_ADDRESS'
+TO_ADDRESS <- '$METALP_TO_ADDRESS'
+EOF
+
+# Delete any database dump config file and recreate from ENV vars
+rm -f .my.cnf
```
Comment (sbancal) [inline app/start.sh]: same comment as line 8

**Review by sbancal — APPROVED** [review]
LGTM with just the 2 tiny comments.

Haven't tested ... Do you need me to test it as well ?

**Comment by evanjt** [comment]
> LGTM with just the 2 tiny comments.
> 
> Haven't tested ... Do you need me to test it as well ?

I don't think it is necessary (building the Dockerfile is already checking by the Github Actions), as the functionality of the application components would be best validated by the lab.

**Review by sbancal — APPROVED** [review]
👍🏼

**Pull request merged**